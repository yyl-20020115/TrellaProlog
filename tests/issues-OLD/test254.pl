:-initialization(main).

main :-
	X is 1 =/= 2.

