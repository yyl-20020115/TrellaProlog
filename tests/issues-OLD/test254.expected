Error: syntax error, near '=/=', operator expected, tests/issues-OLD/test254.pl:4
